test/jdk/sun/net/www/http/KeepAliveCache/B8291637.java

```diff
@@ -125,20 +130,26 @@ public void run() {
         }
     }
 
-    public static void main(String[] args) throws Exception {
-        Server server = new Server(args[0]);
+    public static void runTest(String param) throws Exception {
+        Server server = new Server(param);
         int port = server.getPort();
         server.start();
-        URL url = new URL("http://127.0.0.1:" + Integer.toString(port) + "/");
+        URL url = URIBuilder.newBuilder()
+                .scheme("http")
+                .loopback()
+                .port(port)
+                .path("/firstCall")
+                .toURL();
```

Comment:
Hello Christoph, what's the significance of changing the path from `/` to `/firstCall` in this test?

Reply:
Good catch, copy&paste error.